frontend/src/assets/stylesheets/components/footer.sass

```diff
@@ -0,0 +1,73 @@
+.footer
+  padding: 30px 0px 20px
+  margin-right: 40px
+
+  a
+    color: #A0A0A0
+    text-decoration: none
+    &:hover
+      color: #48bd66
+
+  .social-links
```

Comment:
That's not very BEMish :)

Reply:
Yea, you are right, should use "footer_social-links"

Reply:
I think now that can't be the problem since social-media is block not just elementy any more

Reply:
Ok, it should be moved to a different sass file than.

Reply:
Okey